requirements2.txt

```diff
@@ -0,0 +1,6 @@
+requests >= 2.24.0,<= 2.27.1
+PyYAML >= 5.3.1,<= 5.4.1
+pynuodb >= 2.5.1
+#pynuoadmin >= 2.0.0
```

Comment:
Why?

Reply:
The latest pynuoadmin is 2.1.0 and supports both Python 2 and Python 3. Is that not usable for some reason?

Reply:
see previous comment

Reply:
I was wrong about 2.1.0 supporting Python 3. A new version of pynuoadmin (2.2.0) will be published that supports Python 3 (>=3.6) and 2.7.